### PR TITLE
Updates to the README.md

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,35 @@
 # scylla-cqlsh
 
-A fork of the cqlsh from https://github.com/apache/cassandra
+Command line tool to connect to [scylladb](http://www.scylladb.com) (or Cassandra)
```
Comment (tzach) [inline README.md:3]:
```suggestion
Command line tool to connect to [scylladb](http://www.scylladb.com) (or Apache Cassandra)
```

**Comment by tzach** [comment]
- Should include wording on Scylla Cloud connectivity (can be added later)
- Need to use the same license as upstream (Apache Cassandra)

File: README.md (hunk 1 above)
Comment (fruch) [reply README.md]: fixed

**Comment by fruch** [comment]
> * Should include wording on Scylla Cloud connectivity (can be added later)
> * Need to use the same license as upstream (Apache Cassandra)

Added license information

**Comment by tzach** [comment]
LGTM

**Pull request merged**